### ISSUE TITLE
Can't Transfer Non-EOS Tokens

### ISSUE THREAD

**Issue by joshkauffman** [eoscanada/eosc#94]
https://github.com/eoscanada/eosc/blob/9233f7b05e0b490009c2ba153af830d3b09a6e6e/eosc/cmd/transfer.go#L20

Not able to specify tokens for transfer that are not named EOS
@abourget @fproulx-eoscanada

**Comment by abourget** [comment]
true, should default to EOS, but use what is specified if specified.. it's not what `NewEOSAssetFromString` does, doesn't it?

**Comment by joshkauffman** [comment]
To be honest I'm not sure what that line does, Francois had just told me that it was the one at fault :P

**Comment by DenisCarriere** [comment]
Could we also add support for `delegatebw` & `undelegatebw`

```
ERROR: "network bw stake qty" invalid: EOS has only 4 decimals
```

**Comment by abourget** [comment]
This was solved in v1.1.1
